chapter_recurrent-neural-networks/beam-search.md

```diff
@@ -1,32 +1,33 @@
 # Beam Search
 
-In :numref:`sec_seq2seq`, we discussed how to train an encoder-decoder with input and output sequences that are both of variable length. In this section, we are going to introduce how to use the encoder-decoder to predict sequences of variable length.
+In :numref:`sec_seq2seq`, we discussed how to train an encoder-decoder with input and output sequences that are both of variable length. In this section, we are going to introduce how to use the encoder-decoder to predict sequences of the variable length.
```

Comment:
no need to add "the". Even if "the" is added here, why still keep "output sequences that are both of variable length." (in the previous sentence)